terraform/bastion/Jenkinsfile-check

```diff
@@ -0,0 +1,54 @@
+library("tdr-jenkinslib")
+def repo = "tdr-scripts"
+
+pipeline {
+  agent {
+    label "master"
+  }
+  parameters {
+    choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are checking the age of")
+  }
+  stages {
+    stage("Run git secrets") {
+      steps {
+        script {
+          tdr.runGitSecrets(repo)
+        }
+      }
+    }
+    stage('Destroy bastion if the instance is older than 7 days') {
+      agent {
+        ecs {
+          inheritFrom 'aws'
+          taskrole "arn:aws:iam::${env.MANAGEMENT_ACCOUNT}:role/TDRJenkinsRunEC2DescribeInstances${params.STAGE.capitalize()}"
+        }
+      }
+      stages {
+        stage('Run delete user script') {
+          steps {
+            script {
+              def accountNumber = tdr.getAccountNumberFromStage(params.STAGE)
+              def tooOld = sh(script: "python3 /check_ec2_instance_age.py ${accountNumber} ${params.STAGE} bastion-ec2-instance-${params.STAGE} 7", returnStdout: true).trim()
+              if(tooOld == "True") {
+                build(
+                    job: "TDR Bastion Deploy",
+                    parameters: [
+                        string(name: "STAGE", value: params.STAGE),
+                        string(name: "COMMAND", value: "destroy")
+                    ],
+                    wait: false)
```

Comment:
Could this send a Slack announcement when it succeeds or fails?

Reply:
Yep, makes sense. I've added in a post step.